### ISSUE TITLE
publish results on occasion to github pages

### ISSUE THREAD

**Issue by codefromthecrypt** [wuhuizuo/go-wasm-go#29]
It seems many webassembly libraries claim they are the fastest out there, but someone must be wrong :D I think your project does a decent job looking at different types of use cases where one project may be better suited than another. Even that could change over time as versions become faster within a project.

As you notice in your justfile, some webassembly libraries are harder than others to use, worst being WasmEdge, which requires offline installation or the whole project can't work.

I wonder if you'd be interested in publishing results of a subset of comparisons on some frequency, perhaps to github pages. That would make it easier to validate claims where some project says they are fastest, but aren't. It also can help projects be more competitive, ex working harder to be faster vs just saying they are.

What do you think?

**Comment by codefromthecrypt** [comment]
another way could be to make a fake release in a cron workflow. ex every week publish a release 2022-05-16, etc,  using `gh` commands which have the text of the release as latest comparisons. 

So in this case the releases page is hijacked as a log of performance comparisons!

**Comment by wuhuizuo** [comment]
@codefromthecrypt 
Good idea! The repo is not complete yet, reporter should be quantized. I will try my best to fill the tests and setup pipeline to release the test reports in github pages.

Welcome to contribute!
